etc/config.xml

```diff
@@ -55,5 +55,77 @@
                 <stockprice>0 8 * * *</stockprice>
             </cron>
         </sinchimport>
+        <csp>
+            <mode>
+                <storefront>
+                    <report_only>1</report_only>
+                </storefront>
+                <admin>
+                    <report_only>1</report_only>
```

Comment:
Ditto the above, though I'm not sure that admin enforces anywhere at the moment anyway